src/main/java/org/parabot/core/Core.java

```diff
@@ -239,23 +239,17 @@ public static boolean isValid() {
     }
 
     public enum LaunchMode {
-        INCLUDE_LOCAL("Include local", "-withlocal"),
```

Comment:
What if we would keep this argument and remove the `-`, but use it in the classes; see my example in `Local.java`.